main.js

```diff
@@ -18,8 +18,10 @@
  *        MIT License. All code unless otherwise specified is
  *        Copyright (c) Matthew Knox and Contributors 2015.
  */
-
+'use strict';
 // Load NodeJS Modifications/Variables
+require('babel-register');
+require('babel-polyfill');
```

Comment:
Fine by me, require.js should not be extended (if possible removed) and prototypes.js should be the very first thing subsequently loaded anyway